### PR TITLE
Add missing metrics tags for activity and decision task processing

### PR DESCRIPTION
**Pull request by meiliang86**

### CODE REVIEW

File: src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java [hunk 1]
```diff
@@ -157,26 +160,31 @@ private TaskHandlerImpl(ActivityTaskHandler handler) {
 
     @Override
     public void handle(MeasurableActivityTask task) throws Exception {
-      options
-          .getMetricsScope()
+      Scope metricsScope =
+          options
+              .getMetricsScope()
+              .tagged(
+                  ImmutableMap.of(MetricsTag.ACTIVITY_TYPE, task.task.getActivityType().getName()));
```
Comment (mfateev) [inline src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java]: Why is it not using the activity type LoggerTag on the line 176?

Comment (meiliang86) [reply src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java]: Metrics and logging tag names are different.

Comment (shail) [inline src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java]: Can we also add the WORKFLOW_TYPE metrics tag?

Comment (meiliang86) [reply src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java]: Activity worker does not have that info. Workflow worker has that in decision task completion.

**Pull request merged**